### ISSUE TITLE
Easily Run Sample

### ISSUE THREAD

**Issue by ambroselittle** [Infragistics/samples#7]
As a sample consumer, I want to be able to run samples with one click or less so that I can easily see what they do and ensure it is something like what I want to do.

**Comment by gmurray81** [comment]
Had an interesting idea here. If we had some rolling builds on the repository + some post processing. There could be a well known drop location where one could pick up the runnable samples (sans dependencies), even built against multiple versions, if necessary.

Then the contributor workflow could actually be to submit the sample to the repo, when it is working, then check back a short while later to pick up a runnable version zip, which is already sanitized for the sample consumer, and contains a zip of the code also.

TL;DR; regardless of how the repo is organized, much post processing could be done to make each individual sample as digestible as possible for the consumer. And if it reduces the friction usually involved in sanitizing/staging a sample to send to the consumer, then it goes a lot way to reducing the friction to altering the workflow such that this repo is the system of record for the samples.
